src/app/components/MainContent/index.stories.jsx

```diff
@@ -7,10 +7,34 @@ const data = {
     {
       type: 'headline',
       blockId: '1',
+      model: {
```

Comment:
I think this is ok here, we should probably open an issue to factor out the need to do this so manually. We should be able to generate these using fake-props or autogen-props (remembering those libraries from memory so name may not be exact)

Reply:
Agree